### PR TITLE
Updated new values to Spanish International

### PR DESCRIPTION
**Pull request by samforo**

### CODE REVIEW

File: brevent/src/main/res/values-es/op.xml [hunk 1]
```diff
@@ -19,11 +19,11 @@
 */
```
Comment (liudongmiao) [inline brevent/src/main/res/values-es/op.xml]: don't translate. it's from aosp.

File: brevent/src/main/res/values-es/op_cm.xml [hunk 2]
```diff
@@ -14,11 +14,11 @@
 -->
```
Comment (liudongmiao) [inline brevent/src/main/res/values-es/op_cm.xml]: don't translate, it's from lineageos.

**Review by liudongmiao — CHANGES_REQUESTED** [review]

File: brevent/src/main/res/values-es/op.xml (hunk 1 above)
Comment (samforo) [reply brevent/src/main/res/values-es/op.xml]: ok, but these expressions aren't correct!

File: brevent/src/main/res/values-es/op_cm.xml (hunk 2 above)
Comment (samforo) [reply brevent/src/main/res/values-es/op_cm.xml]: ok, but these expressions aren't correct!

Comment (liudongmiao) [reply brevent/src/main/res/values-es/op_cm.xml]: Then, please pull request to lineage os...

https://github.com/LineageOS/android_packages_apps_Settings/blob/cm-14.1/res/values-es/cm_strings.xml#L130

File: brevent/src/main/res/values-es/op.xml (hunk 1 above)
Comment (liudongmiao) [reply brevent/src/main/res/values-es/op.xml]: Please feedback to aosp.

Here is the mirror of aosp: https://github.com/aosp-mirror/platform_packages_apps_settings/blob/master/res/values-es/arrays.xml#L239

**Comment by liudongmiao** [comment]
When you finish this, I will release a new version tonight.

File: brevent/src/main/res/values-es/op.xml (hunk 1 above)
Comment (samforo) [reply brevent/src/main/res/values-es/op.xml]: Pull in AOSP, Updated translations #17

https://github.com/aosp-mirror/platform_packages_apps_settings/pull/17

File: brevent/src/main/res/values-es/op_cm.xml (hunk 2 above)
Comment (samforo) [reply brevent/src/main/res/values-es/op_cm.xml]: Pull in LineageOS, Updated translations #12

https://github.com/LineageOS/android_packages_apps_Settings/pull/12

File: brevent/src/main/res/values-es/op.xml (hunk 1 above)
Comment (liudongmiao) [reply brevent/src/main/res/values-es/op.xml]: It's a mirror, don't pull request.

if you want to contribute to aosp, follow aosp's instructions. 
Don't contribute to mirror of aosp.

File: brevent/src/main/res/values-es/op_cm.xml (hunk 2 above)
Comment (liudongmiao) [reply brevent/src/main/res/values-es/op_cm.xml]: And, again. I think lineageos use gerrit like aosp too.
Don't pull request on github.

File: brevent/src/main/res/values-es-v26/strings.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-    <string name="brevent_about_developer_adb">No desconecte \"Depuración por USB\", no cambie el \"configuratión de USB\"</string>
+    <string name="brevent_about_developer_adb">No desconecte \"Depuración por USB\", no cambie la \"configuración USB\"</string>
```
Comment (liudongmiao) [inline brevent/src/main/res/values-es-v26/strings.xml:4]: https://github.com/aosp-mirror/platform_frameworks_base/blob/master/packages/SettingsLib/res/values-es/strings.xml#L215

Comment (samforo) [reply brevent/src/main/res/values-es-v26/strings.xml:4]: I can't repair this, I can't edit this string. I don't know how is the problem...

**Pull request merged**